roadmap/messaging-antispam/README.md

```diff
@@ -0,0 +1,34 @@
+# Messaging - Anti-spam
+
+Ravencoin will display messages for all tokens that you own.  But wait, what if I get an unsolicited HERPES token because they found my crypto address on-chain?  Maybe I don’t want messages from the HERPES token issuer.
+
+The holder of the owner token, e.g. ATOKEN! can send messages to holders their token ATOKEN.   The owner token has an exclamation point.  The owner token holder can also create channel tokens like ATOKEN~News, or ATOKEN~VoteInfo, or ATOKEN~Emergency   These are message channel tokens and the holder of these tokens can broadcast a message by attaching a special file (IPFS) to the transaction while sending the message channel token to themselves.
```

Comment:
These tildes are resulting in some weird text formatting..

Reply:
Fixed.  Tilde needed to be escaped.